tutorcredentials/plugin.py

```diff
@@ -19,78 +19,67 @@
 # CONFIGURATION
 ########################################
 
+config: t.Dict[str, t.Dict[str, t.Any]] = {
+    "defaults": {
+        "VERSION": __version__,
+        "BACKEND_SERVICE_EDX_OAUTH2_PROVIDER_URL": "http://lms:8000/oauth2",
+        "BACKEND_SERVICE_EDX_OAUTH2_KEY": "{{ CREDENTIALS_OAUTH2_KEY }}",
+        "DOCKER_IMAGE": "{{ DOCKER_REGISTRY }}overhangio/openedx-credentials:{{ CREDENTIALS_VERSION }}",
+        "EXTRA_PIP_REQUIREMENTS": [],
+        "FAVICON_URL": "https://edx-cdn.org/v3/default/favicon.ico",
+        "HOST": "credentials.{{ LMS_HOST }}",
+        "LOGO_TRADEMARK_URL": "https://edx-cdn.org/v3/default/logo-trademark.svg",
+        "LOGO_TRADEMARK_URL_PNG": "https://edx-cdn.org/v3/default/logo-trademark.png",
+        "LOGO_TRADEMARK_URL_SVG": "https://edx-cdn.org/v3/default/logo-trademark.svg",
+        "LOGO_URL": "",
+        "LOGO_URL_PNG": "{{ CREDENTIALS_LOGO_URL }}",
+        "LOGO_URL_SVG": "",
+        "LOGO_WHITE_URL": "{{ CREDENTIALS_LOGO_URL }}",
+        "LOGO_WHITE_URL_PNG": "{{ CREDENTIALS_LOGO_URL }}",
+        "LOGO_WHITE_URL_SVG": "",
+        "MYSQL_DATABASE": "credentials",
+        "MYSQL_USERNAME": "credentials",
+        "OAUTH2_KEY": "credentials-key",
+        "OAUTH2_KEY_DEV": "credentials-key-dev",
+        "OAUTH2_KEY_SSO": "credentials-key-sso",
+        "OAUTH2_KEY_SSO_DEV": "credentials-key-sso-dev",
+        "PLATFORM_NAME": "{{ PLATFORM_NAME }}",
+        "PRIVACY_POLICY_URL": "{{ LMS_HOST }}/privacy-policy",
+        "SITE_NAME": "{{ LMS_HOST }}",
+        "SOCIAL_AUTH_REDIRECT_IS_HTTPS": False,
+        "SOCIAL_AUTH_EDX_OAUTH2_ISSUER": "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ LMS_HOST }}",
+        "SOCIAL_AUTH_EDX_OAUTH2_URL_ROOT": "http://lms:8000",
+        "SOCIAL_AUTH_EDX_OAUTH2_KEY": "credentials-sso-key",
+        "SOCIAL_AUTH_EDX_OAUTH2_LOGOUT_URL": "{{ LMS_HOST }}/logout",
+        "THEME_NAME": "edx-theme",
+        "TOS_URL": "{{ LMS_HOST }}/tos",
+    },
+    "unique": {
+        "MYSQL_PASSWORD": "{{ 8|random_string }}",
+        "SOCIAL_AUTH_EDX_OAUTH2_SECRET": "{{ 16|random_string }}",
+        "BACKEND_SERVICE_EDX_OAUTH2_SECRET": "{{ 16|random_string }}",
+        "OAUTH2_SECRET": "{{ 16|random_string }}",
+        "OAUTH2_SECRET_DEV": "{{ 16|random_string }}",
+        "OAUTH2_SECRET_SSO": "{{ 16|random_string }}",
+        "OAUTH2_SECRET_SSO_DEV": "{{ 16|random_string }}",
+    },
```

Comment:
please add an empty key with name `overrides` with a comment in it.
```
"overrides" : {
        # Add values to override settings from Tutor core or other plugins here.
        # Each override is a pair, (setting_name, new_value). For example:
        # ("PLATFORM_NAME", "My platform"),
}
```

Reply:
It's a choice to add it or not to add it.

Reply:
Actually, no other MFE has this empty `overrides` key, so I think it's better to leave it this way.

Reply:
yes, it was a choice anyway.